### ISSUE TITLE
Add server/client version check

### ISSUE THREAD

**Issue by Elmegaard** [molenzwiebel/Wetstone#6]
As my mod is both a client/server mod, I get so many issues from people reporting the mod doesn't work. I was thinking of making a client/server check in both directions for my mod but adding it to Wetstone seems better.

So this should probably be configurable, but basically on load, send a message from client to server with your version and mod name, then the server replies with it's version/name. If nothing is received within x time, write in chat that your mod was not found on the server. If the versions don't match, write that.

Some form of that, if that makes sense.

**Comment by Elmegaard** [comment]
Could do the same with Wetstone for a version check. Like if my mod requires version 1.1.0, but only 1.0.0 is installed, it would also show this error, etc.

**Comment by molenzwiebel** [comment]
Yes, some form of semver version communication would be nice/desirable to have.
